docs/Homebrew-Governance.md

```diff
@@ -0,0 +1,97 @@
+
+## 3. General Meetings of Members
+
+1. A general meeting of the members may be called by the PLC with at least three weeks notice.
+
+2. The quorum to vote on resolutions and elections at a general meeting is 3 voting members or 10% of the voting members, whichever is greater.
+
+3. Homebrew members will meet at the annual general meeting (AGM) in a manner determined by the PLC.
+
+4. General elections will be held at the AGM.
+
+5. The PLC will announce candidates and proposals three weeks prior to the election date.
+
+6. Members may cast a vote any time up to three weeks prior to the election date.
+
+### 3.1. Amendments to these bylaws
+
+1. These bylaws may be amended by a special resolution at a general meeting of the members. A special resolution requires a two-thirds supermajority.
+
+2. Any member may propose an amendment via pull request on GitHub against this document.
+
+3. Members shall vote on any amendments by approving or requesting changes on the GitHub pull request. Voting will close three weeks after an amendment is proposed, and all votes tallied.
+
+4. Any approved amendments will take effect three weeks after the close of voting.
+
+5. No amendment may be proposed which violates Software Freedom Conservancy policies. The PLC may veto such an amendment by an ordinary resolution.
+
+## 4. Project Leadership Committee
+
+1. The governance and operation of Homebrew is determined by the PLC. The PLC will represent Homebrew in all dealings with the Software Freedom Conservancy (SFC). The PLC has final authority in all matters related to the operation of Homebrew, and will comply with all Software Freedom Conservancy policies.
+
+2. The PLC consists of five members. Committee members are elected by majority vote of Homebrew members. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
+
+3. Three weeks prior to the AGM, the PLC will nominate at least one candidate for each upcoming vacant seat. Any member may also nominate any other member as a candidate for the PLC at this time.
+
+4. The PLC will deliver a report of their activities, decisions, and financial statements to the Homebrew members at the AGM.
+
+5. No more than two employees of the same employer may serve on the PLC.
+
+## 5. Meetings of the Project Leadership Committee
+
+1. A meeting of the PLC may be called by any two of its members with at least three weeks notice, unless all PLC members agree to a shorter notice period.
+
+2. The quorum to vote on resolutions at a meeting of the PLC is a majority of its members.
+
+3. A majority of the entire membership of the PLC is required to pass an ordinary resolution. A two-thirds supermajority of the entire membership of the PLC is required to pass a special resolution.
```

Comment:
FYI, there's still no definition of what a special resolution of the PLC is. All business mentioned here appears to be normal resolutions.

Reply:
You're right. There are no special resolutions of the PLC. This sentence can be removed.

Reply:
```suggestion
3. A majority of the entire membership of the PLC is required to pass an ordinary resolution.
```

Reply:
This is referenced/defined above: 

> 1. These bylaws may be amended by a special resolution at a general meeting of the members. A special resolution requires a two-thirds supermajority.

Reply:
so my understanding is: the only special resolution is a bylaw amendment. Perhaps `special resolution` could be replaced with `bylaw amendment` in here?

Reply:
@MikeMcQuaid yes, a bylaw amendment is a special resolution, but, of *the entire membership*. The PLC cannot and should not be able to amend the bylaws on their own (IMO). The definition of special resolution and bylaw amendment for members is already defined elsewhere in the document.

Reply:
Gotcha.